src/crawler/crawl.js

```diff
@@ -3,106 +3,165 @@ require("dotenv").config({ path: path.join(__dirname, "../../.env") });
+    return Array.from(elements)
+      .map((element) => element.href)
+      .slice(0, 5);
+  });
+
+  for (const link of links) {
+    const videoData = await Video.findOne({
+      youtubeVideoId: link.split("=")[1],
+    });
+
+    if (!videoData) {
+      linksQueue.push(link);
+    }
+  }
+
+  const newURL = linksQueue.shift();
+
+  try {
+    newVideoObj.title = await page.$eval(
+      TITLE_SELECTOR,
+      (element) => element.textContent,
     );
   } catch (error) {
     console.error(error);
   }
 
   try {
-    await page.$eval(
-      "#primary-button > ytd-button-renderer > yt-button-shape > button > yt-touch-feedback-shape > div > div.yt-spec-touch-feedback-shape__fill",
-      (button) => button.click(),
+    newVideoObj.description = await page.$eval(
+      DESCRIPTION_SELECTOR,
+      (element) => element.textContent,
     );
-    await page.waitForSelector(
-      "#segments-container > ytd-transcript-segment-renderer yt-formatted-string",
+  } catch (error) {
+    console.error(error);
+  }
+
+  try {
+    newVideoObj.channel = await page.$eval(
+      CHANNEL_SELECTOR,
+      (element) => element.textContent,
     );
   } catch (error) {
     console.error(error);
   }
 
-  const title = await page.$eval(
-    "#title > h1 > yt-formatted-string",
-    (el) => el.textContent,
-  );
-  const description = await page.$eval(
-    "#description-inline-expander > yt-attributed-string > span > span:nth-child(1)",
-    (el) => el.textContent,
-  );
-  const channel = await page.$eval("#text > a", (el) => el.textContent);
-  const transcripts = await page.$$eval(
-    "#segments-container > ytd-transcript-segment-renderer yt-formatted-string",
-    (elements) => elements.map((el) => el.textContent),
+  const transcripts = await page.$$eval(TRANSCRIPT_SELECTOR, (elements) =>
+    elements.map((element) => element.textContent),
   );
-  const transcript = transcripts.join(" ");
-  const metaTags = await page.$$eval("meta", (elements) => {
+
+  newVideoObj.transcript = transcripts.join(" ");
+
+  const metaTags = await page.$$eval(META_SELECTOR, (elements) => {
     const result = { thumbnailURL: "", tag: "" };
 
-    elements.forEach((el) => {
-      const property = el.getAttribute("property");
-      const name = el.getAttribute("name");
+    elements.forEach((element) => {
+      const property = element.getAttribute("property");
+      const name = element.getAttribute("name");
 
       if (property === "og:image") {
-        result.thumbnailURL = el.getAttribute("content");
+        result.thumbnailURL = element.getAttribute("content");
       }
 
       if (name === "keywords") {
-        result.tag = el.getAttribute("content");
+        result.tag = element.getAttribute("content");
       }
     });
 
     return result;
   });
 
-  const { thumbnailURL } = metaTags;
-  let { tag } = metaTags;
+  newVideoObj.thumbnailURL = metaTags.thumbnailURL;
+  newVideoObj.tag = metaTags.tag;
```

Comment:
```suggestion
  newVideoObject.thumbnailURL = metaTags.thumbnailURL;
  newVideoObject.tag = metaTags.tag;
```
이부분도 줄임말 대신 풀네임으로 쓰면 좋을 것 같습니다